deployer/templates/es.yaml

```diff
@@ -92,7 +96,7 @@ objects:
                   mountPath: /etc/elasticsearch/secret
                   readOnly: true
                 - name: elasticsearch-config
-                  mountPath: /usr/share/elasticsearch/config
+                  mountPath: /etc/elasticsearch/config
```

Comment:
the changes in the 'testing' commit seem to work around all that wackiness i was doing before...